src/systems/filecoin_vm/actor/actor.go

```diff
@@ -6,6 +6,8 @@ const (
 	MethodSend        = MethodNum(0)
 	MethodConstructor = MethodNum(1)
 	MethodCron        = MethodNum(2)
+
+	MethodGetUnsealedCIDForDealIDs = MethodNum(99)
```

Comment:
we prob dont need this here, this should prob live in the same file as the `StorageMarketActor` code?

Reply:
Won't it be hard to track `MethodNum` duplication if we don't have a centralized registry lexically in the code?

Reply:
They don't need to be globally unique, only scoped to each actor.

Reply:
I see, thanks. I'm not up to speed so appreciate the tutorial.

Reply:
(@porcuquine Although, they do need to avoid colliding with the reserved constants 0, 1, 2.)

Reply:
Yeah, I chose 3 for that reason: `	MethodGetUnsealedCIDForDealIDs = actor.MethodNum(3)`.